7_react/react1/week1/vite-project/src/App.jsx

```diff
@@ -0,0 +1,37 @@
+import IteratingTodoItemsArray from "./components/iteratingTodoItemsArray";
+import "./App.css";
+
+const toDoItems = [
+  {
+    id: "1",
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017",
+  },
+
+  {
+    id: "2",
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017",
+  },
+
+  {
+    id: "3",
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017",
+  },
+];
+
+function App() {
+  return (
+    <>
```

Comment:
this Fragment `<>` is unnecessary, you can remove it :)

Reply:
Hey @BrantalikP  
Thank you so much. 
Done :)